mobile/src/people/people-department.tsx

```diff
@@ -4,17 +4,17 @@ import { connect, Dispatch } from 'react-redux';
 
 import { EmployeesList } from './employees-list';
 import { AppState } from '../reducers/app.reducer';
-import { EmployeeMap } from '../reducers/organization/employees.reducer';
+import { EmployeeMap, EmployeesStore } from '../reducers/organization/employees.reducer';
 import { Employee } from '../reducers/organization/employee.model';
 import { openEmployeeDetailsAction } from '../employee-details/employee-details-dispatcher';
 
 interface PeopleDepartmentProps {
-    employeesMap: EmployeeMap;
+    employees: EmployeesStore;
```

Comment:
what you should have instead is employeesByDepartments reference, when you can compare them using equals

Reply:
had discussion, agreed to leave it as is